package.json

```diff
@@ -29,13 +29,16 @@
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.9.0",
     "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.16.10",
     "@babel/preset-env": "^7.9.5",
     "@babel/register": "^7.9.0",
+    "@babel/runtime": "^7.16.7",
     "mocha": "^7.1.1"
   },
   "babel": {
     "plugins": [
-      "@babel/plugin-proposal-class-properties"
+      "@babel/plugin-proposal-class-properties",
+      "@babel/transform-runtime"
```

Comment:
@paribaker - Why is this new transform needed?

Reply:
I came across this error when trying to run tests after a couple of posts around the topic I managed to get it to work from the babel docs

https://stackoverflow.com/questions/57219314/error-while-trying-to-run-scripts-regeneratorruntime-is-not-defined